lib/improver/spotdata/read_input.py

```diff
@@ -130,28 +138,42 @@ def get_additional_diagnostics(diagnostic_name, diagnostic_data_path,
     """
     Load additional diagnostics needed for particular spot data processes.
 
-    Args
-    ----
+    Args:
+    -----
     diagnostic_name : The name of the diagnostic to be loaded. Used to find
                       the relevant file.
 
     time_extract    : An iris constraint to extract and return only data from
                       the desired time.
 
-    Returns
-    -------
+    Returns:
+    --------
     cube            : An iris.cube.CubeList containing the desired diagnostic
-                      data, with a single entry is time_extract is provided.
+                      data, with a single entry if time_extract is provided.
```

Comment:
I have avoided merging due to the issues that inconsistent metadata (even things like history?) can throw up and which must be handled. I don't actually need this to be a cube, a cubelist is fine. We have now factored out the cube concatentation that Gavin wrote, so I could use that if you feel strongly?